assets/_scss/default/body-main-styles.scss

```diff
@@ -206,8 +208,93 @@ $border-color: #a1a1a1;
 .services-item.active .services-item__content {
   display: block;
 }
-#title{
+#link {
+  color: #000000;
+  text-decoration: underline;
+  font-style: italic;
+  font-size: 2.4rem;
+}
+#center {
+  display: flex;
+  justify-content: center;
+}
+#titleWWU {
+  color: rgb(80, 80, 80);
   font-size: 50px;
+}
+.containertitle {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  text-align: center;
+  width: 65%;
+  margin: 0 auto;
+}
+#imagename {
+  width: 700px;
+  margin: 0 auto;
+}
+#title {
+  font-size: 80px;
+  margin: 0 auto;
+  color: var(--title-color);
   text-align: center;
+}
+#title2 {
+  color: var(--title-color);
+  font-size: 50px;
+}
+.containerCompany {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
   margin: 0 auto;
 }
+.boxCompany {
+  width: 95%;
+  text-align: left;
+  font-size: 20px;
+}
+.containerTeam {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+.containerMember {
+  width: 30%;
+  text-align: center;
+  margin-bottom: 20px;
+}
+#curating {
+  text-align: left;
+  font-weight: 300;
+  font-size: 2.4rem;
+}
+.imgMembers{
+  height: 400px;
+  width: 350px;
+  object-fit: cover;
+  box-shadow: 0 8px 8px 0 rgba(0, 0, 0, 0.15);
+}
+#imgMember{
+  border-radius: 50% 20% / 10% 40%;
+}
+#imgMember2 {
+  border-radius: 20% 50% / 40% 10%;
+}
+#imgMember3 {
+  border-radius: 25% 15%;
+}
+#imgMember4 {
+  border-radius: 15% 25%;
+}
+#desc {
+  font-size: 19px;
+  font-weight: 900;
+  text-align: left;
+}
+.job {
+  font-size: 2rem;
+  font-weight: 900;
+  color: #756f6f;
```

Comment:
If this color will could be reused in the future, I would also add it to a variable